python/arcticdb/version_store/_normalization.py

```diff
@@ -431,12 +436,14 @@ def _normalize_columns_names(columns_names, index_names, norm_meta, dynamic_sche
         else:
             if dynamic_schema and (counter[col] > 1):
                 raise ArcticNativeException("Same column names not allowed in dynamic_schema")
+            new_name = col_str
             if counter[col] > 1 or col in index_names:
                 new_name = "__col_{}__{}".format(col, 0 if dynamic_schema else idx)
+            if not isinstance(col, str) or new_name != col_str:
                 norm_meta.common.col_names[new_name].original_name = col_str
-                if isinstance(col, int):
-                    norm_meta.common.col_names[new_name].is_int = True
```

Comment:
Yeah I will bring it back

Reply:
Yeah, I will bring it back

Reply:
Thanks, it would be good to quickly test the interop here, ie can data written by this client be read by older client and vice versa, can data written by older client be read by this client?